api/migrations/versions/28da946c5e79_.py

```diff
@@ -79,7 +79,8 @@ def upgrade():
     sa.Column('name', sa.String(length=1024), nullable=True),
     sa.Column('state', sa.String(length=15), nullable=True),
     sa.Column('choice', sa.Integer(), nullable=True),
-    sa.Column('consumption_date', sa.DateTime(), nullable=True),
+    sa.Column('consumption_date', sa.DateTime(), nullable=True),	
+	sa.Column('corp_num', sa.String(length=10), nullable=True),
     sa.Column('remote_name_id', sa.BigInteger(), nullable=True),
```

Comment:
Hi Steven, this is a migration script, meaning it's called one time (upon deployment) to make changes to the database. This was already called in the beginning of the project. So it will not be re-run, and we do not add to them. So please remove this code change, and I can show you how to create a new migration script - it's a python thing.